docs/HACKING.md

```diff
@@ -12,6 +12,17 @@ The operator image needs to be pushed to a remote repository:
 ```shell script
 podman push quay.io/<insert username>/wmco:$VERSION_TAG
 ```
+
+Building the full WMCO image is a time consuming process given we are building all the submodules. If you are just
+changing WMCO and not the modules, you can separately build the submodules as a base image and then just build the WMCO
+image from this base image:
+```shell
+podman build . -t wmco-base:latest -f build/Dockerfile.base
+podman build . -t quay.io/<insert username>/wmco:$VERSION_TAG -f build/Dockerfile.wmco
+podman push quay.io/<insert username>/wmco:$VERSION_TAG
```

Comment:
It would be nice if a make target was added that did this.
So then you could just do something like this:
`make dev-build IMG=quay.io/<insert username>/wmco:$VERSION_TAG`

Reply:
```
.PHONY: dev-build
dev-build:
        podman build . -t wmco-base:latest -f build/Dockerfile.base
        podman build . -t $(IMG) -f build/Dockerfile.wmco
        podman push $(IMG)
```

Reply:
The main reason for the split Dockerfiles was so that you don't keep rebuilding the submodules which is exactly what this `Makefile` endpoint you are asking for is doing. IOW the endpoint is as good as doing `podman build . -t $(IMG) -f build/Dockerfile; podman push $(IMG)`. So I don't see how it helps us. So if you want `Makefile` endpoints it would be `dev-base-build`, `dev-wmco-build`, `dev-img-push`. Alternatively looking more at the new `Makefile`, I am wondering if `make run` with a local `/payload` directory is good enough. At which point, what we should do is add a `payload` endpoint.

Reply:
>I am wondering if make run with a local /payload directory is good enough. At which point, what we should do is add a payload endpoint.

Just remembered that this won't work for us as WMCO needs to run on the private subnet.